### DISCUSSION TITLE
Where are categories stored?

### DISCUSSION THREAD

**Discussion by pungentstentch** [Heroic-Games-Launcher/HeroicGamesLauncher#3282] [Q&A] [answered]
I'd love to be able to save and move the categories once I'd spent time curating them.
Since I'm using flatpak I'd  checked ~/.var/app/com.heroicgameslauncher.hgl/
but can't find it.

**Comment by arfonso01** [comment, marked as answer]
hi, for custom categories check '.var/app/com.heroicgameslauncher.hgl/config/heroic/store/config.json' and  if you want to generate the categories automatically, look at this: [Auto Heroic Categories](https://github.com/arfonso01/auto-heroic-categories)

**Comment by scwoodson** [comment]
Hello! Where would custom categories be stored for the Windows version of Heroic (version 2.15.1)?
I'm trying to update categories for all of my games on Steam Deck, but it's significantly easier to do it on Windows. I'm hoping that I can find the config.json on my Windows Heroic launcher, and copy/paste that to my steam deck.

Thanks for your time and help!
-Stevo

**Reply by SPYderHASS** [reply]
Hi scwoodson,
I've just the same question and end up here. Once I know the name of the file (config.json) I start to search for it in my user folder (with Total Commander) and voila, there is the location: 

* C:\\Users\\[yourUserName]\\AppData\\Roaming\\heroic\\store\config.json

I can't say it for sure because I didn't tried it yet: But be careful when replacing the entire Windows-file with the SD-file version. Do a backup 1st. I expect it would be a better idea to just do a merge (cut/replace) of only the "customCategories" section of the config.json over an entire replacement of the file.

**Comment by kyzorr** [comment]
the Flatpak installation unter PikaPS (Debian) use the dir:

- /home/user/.config/heroic/
- /home/user/.config/heroic/store
- /home/user/.config/heroic/store_cache

for the configs
